web/src/settings_config.ts

```diff
@@ -561,6 +561,9 @@ export const preferences_settings_labels = {
     ),
     fluid_layout_width: $t({defaultMessage: "Use full width on wide screens"}),
     high_contrast_mode: $t({defaultMessage: "High contrast mode"}),
+    realm_enter_sends: new Handlebars.SafeString(
```

Comment:
I think it should just be `enter_sends`, since we use `realm_` prefix for the new user defaults.

Reply:
Done.